src/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -382,15 +382,14 @@ public void Timeout_TooShort_AllPendingOperationsCanceled()
 
         [Fact]
         [OuterLoop]
-        [ActiveIssue(12778)]
         public void Timeout_SetTo60AndGetResponseFromServerWhichTakes40_Success()
         {
             // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
             const string SlowServer = "http://httpbin.org/drip?numbytes=1&duration=1&delay=40&code=200";
             
             using (var client = new HttpClient())
             {
-                client.Timeout = TimeSpan.FromSeconds(60);
+                client.Timeout = TimeSpan.FromMinutes(5);
```

Comment:
You can't change this timeout value without also changing the SlowServer URL so that it behaves in the updated manner.

The reason why this test is written the way it is, is because the current URL (the querystring parameters actually) cause this special test server to delay sending a response for 40 seconds.  That is why the `delay=40` is in the URL.  So, we set the HttpClient timeout to be greater than that (60 seconds) and thus we are testing that we are successful in getting a response under 60 seconds (since the server only delays 40 seconds).

If you want to have a longer timeout, then you need to change the URL querystring parameters accordingly.  And I'm not sure that httpbin.org even supports that long timeout.